modern/src/reports/StopReportPage.js

```diff
@@ -37,43 +40,43 @@ const StopReportPage = () => {
     headerName: t('reportStartTime'),
     field: 'startTime',
     type: 'dateTime',
-    flex: 1,
+    width: theme.dimensions.dateColumnWidth,
```

Comment:
Not sure I'm very happy with fixed width for columns, but we can leave it as is for now.

Reply:
ok.